apps/src/templates/sectionProgressV2/SectionProgressV2.jsx

```diff
@@ -17,12 +17,13 @@ function SectionProgressV2({
   unitData,
   isLoadingProgress,
   isRefreshingProgress,
+  isLevelProgressLoaded,
 }) {
   const [expandedLessonIds, setExpandedLessons] = React.useState([]);
 
   const levelDataInitialized = React.useMemo(() => {
-    return unitData && !isLoadingProgress && !isRefreshingProgress;
-  }, [unitData, isLoadingProgress, isRefreshingProgress]);
+    return unitData && isLevelProgressLoaded;
+  }, [unitData, isLevelProgressLoaded]);
```

Comment:
Could the `useEffect` just below also be updated to use `isLevelProgressLoaded` or is there a subtle distinction there?

Reply:
No because we don't want to send off a request if we are already loading. We could add `isLevelProgressLoaded` in addition to the three already there, but it would be redundant